content/realtime/metachannels.textile

```diff
@@ -79,53 +84,171 @@ For further information on capabilities, please refer to the following resources
+- region := The region the request was made in.<br>__Type: @String@__
+- headers := The headers sent with the request.<br>__Type: @JSON Object@__
+- query := The details of the parsed request query, including information such as the request format.<br>__Type: @JSON Object@__
+- path := The path of the endpoint called in the request, for example @/channels/{channel-name}/messages@.<br>__Type: @String@__
+- channels := A list of channels the request was made against.<br>__Type: @Array@__
+- error := The details of any error encountered whilst making the request. It includes an error message, error code and HTTP status code.<br>__Type: @JSON Object@__
+
+An example of a @request.succeeded@ event is:
+
+```[json]
+{
+  "host": "rest.ably.io",
+  "requestId": "fbbcb0ab-fa56-47c4-bbd4-fccc22a271c9",
+  "region": "us-east-2",
+  "headers": {
+    "host": "rest.ably.io",
+    ...
+  },
+  "query": {
+    "format": "json"
+  },
+  "path": "/channels/example-channel/messages",
+  "channels": [
+    "my-test-channel"
+  ]
+}
+```
+
+h2(#reactor). Reactor rules
+
+You can associate "Reactor rules":https://www.ably.io/reactor with metachannels in the same way as for regular channels. Reactor rules are often used with metachannels so that events can be sent to "serverless endpoints":/general/events, "queues":/general/queues, and "streams":/general/firehose. To configure a reactor rule with a metachannel, use the "Message":/realtime/messages source and use a @channelFilter@ of, for example, @^\[meta\]log@. Normally the @channelFilter@ regex will only match normal, non-meta channels; however, a regex of the form @^\[qualifier\]pattern@ is special-cased to match channels with that qualifier. The @pattern@ is a regex as normal.
 
-You can associate "Reactor rules":https://www.ably.com/integrations with metachannels in the same way as for regular channels; configure these using the "Message":/realtime/messages source and use a @channelFilter@ of, for example, @^\[meta\]log@. Normally the @channelFilter@ regex will only match normal, non-meta channels; however, a regex of the form @^\[qualifier\]pattern@ is special-cased to match channels with that qualifier. The @pattern@ is a regex as normal.
+h3(#lifecycle-webhooks). Channel lifecycle outbound webhooks
 
-If you want webhooks for channel lifecycle events, creating a "channel lifecycle webhook":/general/events#sources is an easy alternative to creating a message rule for the @[meta]channel.lifecycle@ metachannel; such webhooks will have simpler POST bodies as the lifecycle events won't be wrapped in "@Message@":/realtime/types#message objects.
+When creating an outbound webhook Reactor rule it is possible to select @Channel Lifecycle@ as the "source":general/events#sources. This will set all channel lifecycle events as triggers for the webhook, rather than having to create individual message rules for each event type.
```

Comment:
> rather than having to create individual message rules for each event type.

That's not a thing you would need to do either way, a message rule gets every message on the channel. The reason to use lifecycle webhooks is the one in the previous version -- simpler and more readable post bodies -- and also that we have the freedom to move lifecycle events to a new mechanism, which we're planning on doing soon.

How about:

`If you're considering creating a message rule on the @[meta]channel.lifecycle@ metachannel, we suggest that you instead create a rule with @channel.lifecycle@ as the "source":general/events#sources. Events from this will be much simpler to parse as the lifecycle events won't be wrapped in "@Message@":/realtime/types#message objects. This is currently only selectable in batched Webhook rules; however we are shortly planning to extend this to all rule types, "get in touch":https://ably.com/contact if you'd be interested in this.`

Reply:
Thanks Simon - reworded this to point that out and removed the individual rule sentence.